tests/integration/test_s3.py

```diff
@@ -436,12 +465,18 @@ def test_s3_get_get_object_headers(self):
 
     def test_s3_head_response_content_length_same_as_upload(self):
         bucket_name = 'test-bucket-%s' % short_uid()
-        self.s3_client.create_bucket(Bucket=bucket_name)
+        client = boto3.client(
```

Comment:
Better extract this into a small util function, to avoid duplication in this file.

Also, let's better use `config.get_edge_url()` for `endpoint_url`.

Reply:
yeah. so I will create one function which will return s3 boto client with TEST credentials.